WordPress/src/test/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModelTest.kt

```diff
@@ -128,6 +131,16 @@ class ActivityLogViewModelTest {
         verify(store, never()).getActivityLogForSite(site, false)
     }
 
+    @Test
+    fun headerIsDisplayedForFirstItemOrWhenDifferentThenPrevious() {
+        val canLoadMore = true
+        viewModel.onActivityLogFetched(OnActivityLogFetched(3, canLoadMore, FETCH_ACTIVITIES))
+
+        Assert.assertTrue(events.last()?.get(0)?.isHeaderVisible(null) ?: false)
```

Comment:
You could do `events.last()?.get(0)?.isHeaderVisible(null) == true` when comparing nullable Boolean values in Kotlin. 
NIT: I'd use static imports for Assert.*